topic/timeseries/dask-weather-data-import.ipynb

```diff
@@ -7,10 +7,19 @@
    "source": [
     "# How to Build Time Series Applications in CrateDB\n",
     "\n",
-    "This notebook guides you through an example of how to import and work with\n",
+    "This notebook guides you through an example of how to batch import \n",
     "time series data in CrateDB. It uses Dask to import data into CrateDB.\n",
     "Dask is a framework to parallelize operations on pandas Dataframes.\n",
     "\n",
+    "## Important Note\n",
+    "If you are running this notebook on a (free) Google Colab environment, you \n",
+    "might not see the parallelized execution of Dask operations due to constrained\n",
+    "CPU availability.\n",
```

Comment:
That's good to know. Thanks for finding out, and for conveying that information to readers and users of the relevant notebook.